buildSrc/build.gradle.kts

```diff
@@ -30,6 +30,11 @@ repositories {
     url = uri("https://repo.spring.io/plugins-release/")
     content { includeGroup("io.spring.gradle") }
   }
+  // For obsolete Avro plugin
+  maven {
+    url = uri("https://jitpack.io")
```

Comment:
Do we still need this?

Reply:
@sjvanrossum kind ping on this question

Reply:
That's what I'm wondering as well. Downloads for this dependency from Gradle's Plugin Portal repository forward to JCenter/Bintray, which is not active but still continues to serve artifacts. The service has gone down a few times since it was sunset in 2021, but if you have plans to rip out this plugin altogether in the near future, then it's not necessary to change this over to JitPack.

Reply:
Perhaps we should keep this PR around in case the build breaks on availability of this plugin again before its fully replaced.